package.json

```
@@ -77,7 +77,7 @@
     "stylelint-config-prettier": "^8.0.2",
     "tslib": "~2.3.0",
     "typescript": "~4.9.5",
-    "zone.js": "~0.11.4"
+    "zone.js": "~0.13.3"
   },
   "peerDependencies": {
     "@angular/common": "13.x || 14.x",
```

I think the peerDeps should be upgraded

```suggestion
    "@angular/common": "15.x || 16.x",
    "@angular/core": "15.x || 16.x",
    "@angular/forms": "15.x || 16.x",
    "rxjs": "7.x"
```